src/utils/types/equipmentType.ts

```diff
@@ -167,8 +174,21 @@ export interface EquipmentInfos extends Identifiable {
     type: EquipmentType;
     voltageLevelLabel?: string;
     voltageLevelId?: string;
+    operatingStatus?: string;
+    terminal1Connected?: boolean;
+    terminal2Connected?: boolean;
+    voltageLevelId1?: UUID;
+    voltageLevelName1?: string;
+    voltageLevelId2?: UUID;
+    voltageLevelName2?: string;
```

Comment:
it's not very consistent that voltageLeveld is a string and voltageLevelId1, voltageLevelId2 are UUID.

Reply:
Done